docs/source/release/v6.14.0/indirect_geometry.rst

```diff
@@ -7,23 +7,34 @@ Indirect Geometry Changes
 
 New Features
 ------------
-.. amalgamate:: Indirect/New_features
+
 
 
 Bugfixes
 --------
-.. amalgamate:: Indirect/Bugfixes
+
 
 
 Algorithms
 ----------
 
 New features
 ############
-.. amalgamate:: Indirect/Algorithms/New_features
+- *<prefix>-phonopy.yml* and *<prefix>-force_constants.hdf5* files
```

Comment:
This could (should?) be combined in a single line - and again on L33? Maybe it doesn't matter when it gets formatted?